src/contracts/krAsset/IKreskoAsset.sol

```diff
@@ -17,11 +19,11 @@ interface IKreskoAsset is IERC20Upgradeable {
 
     function mint(address _to, uint256 _amount) external;
 
-    function rebalance() external view returns (bool expand, uint256 rate);
+    function rebaseInfo() external view returns (Rebase memory);
 
-    function rebalanced() external view returns (bool);
+    function isRebased() external view returns (bool);
 
-    function setRebalance(uint256 _rate, bool _expand) external;
+    function rebase(uint256 _denumerator, bool _positive) external;
```

Comment:
Is the variable name `_denumerator` intentionally meant to be a new word that combines `denominator` and `numerator`?

Reply:
no its a typo 🐛